packages/react-composites/src/composites/CallWithChatComposite/styles/EmbeddedPeoplePane.styles.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles } from '@fluentui/react';
+
+export const copyLinkButtonStyles: IButtonStyles = {
+  root: {
+    height: '3rem',
+    border: 'none',
```

Comment:
Doesn't look like primary buttons have borders that differ from the buttons color: https://developer.microsoft.com/en-us/fluentui#/controls/web/button
Is this needed?